js/core/dom_adapter.js

```diff
@@ -171,8 +176,14 @@ const nativeDOMAdapterStrategy = {
         };
     },
 
-    elementsFromPoint(x, y) {
-        return this._document.elementsFromPoint(x, y);
+    elementsFromPoint(x, y, element) {
```

Comment:
Can you explain please, why should this function get `element` as a parameter?

Reply:
Because element can be inside of shadowRoot. In this case we cannot use the `document.elementsFromPoint` method.